src/table.ts

```diff
@@ -2072,6 +2084,14 @@ function getNextDelay(numConsecutiveErrors: number, config: BackoffSettings) {
   return Math.min(calculatedNextRetryDelay, config.maxRetryDelayMillis);
 }
 
+function populateAttemptHeader(attempt: number, gaxOpts?: CallOptions) {
```

Comment:
Looks like a good approach to me. I think a good TODO eventually would be to create a function for these next three lines of code because we repeat it in `mutateRows` and `readRows`, but we can do that later.

Reply:
This is already extracted to its own function?